apps/prairielearn/src/pages/instructorEffectiveUser/instructorEffectiveUser.js

```diff
@@ -89,31 +90,31 @@ router.post('/', function (req, res, next) {
     res.clearCookie('pl_requested_course_instance_role');
     res.clearCookie('pl_requested_mode');
     res.clearCookie('pl_requested_date');
-    res.cookie('pl_requested_data_changed');
+    res.cookie('pl_requested_data_changed', null);
```

Comment:
Rather than `null`, I'd use a value like `'1'` or `'true'`.

Reply:
This value has been updated to `'true'` in commit [fd97b5c](https://github.com/PrairieLearn/PrairieLearn/pull/9091/commits/fd97b5c1776985f4dba6a11318040ed5a4790d1f).

Reply:
Sorry if this wasn't clear, you should make this change for *all* `res.cookie(...)` calls that you changed to take `null` as a second argument.

Reply:
That makes sense! Sorry, I misunderstood. I have now changed the value for all cookies to be `'true'`.